book/analytics/analytics_utility/analytics_utility.md

```diff
@@ -35,19 +65,7 @@ For example, when you click on the Side Menu, you’ll see:
 
 You may notice that the Data Property field does not have values. That suggests that this particular event does not include any custom properties.
 
-#### iOS
-
-1. Download your app in debug mode
-2. Go to your app in Settings and turn on “Show Analytic Utility Alerts" within the *Applicaster Member Mode* section:
-	![alert_ios_2](./alert_ios_2.png)
-3. Now launch the app
-4. You should now be able to see the alerts:
-	![alert_ios_5](./alert_ios_5.png)
-
-
-## Viewing Logs in the Console
-
-#### Android
+### Viewing Logs in the Console
 
 If you do not have Android Studio, you’ll need to download it. You can do so [here](https://developer.android.com/studio/index.html).
```

Comment:
I think we can delete all the AVD creation. 
Android developers already have it :)

Reply:
This is not necessarily aimed at developers.

Reply:
at least not of the same platform